ionos_dev/search-engine/search/deployment.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: dome-search-deployment
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/instance: dome-search
+      app.kubernetes.io/name: dome-search
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/instance: dome-search
+        app.kubernetes.io/name: dome-search
+    spec:
+      containers:
+        - name: search
+          image: mardetom/search:0.0.2
+          ports:
+            - containerPort: 8080
+              hostPort: 8080
+              protocol: TCP
+          env:
+            - name: ELASTICSEARCH_ADDRESS_HOST
+              value: dome-elasticsearch-svc
```

Comment:
are you sure this is the actual name of the service?

Reply:
Just to be clear, this value must be the dependecies name of Chart.yaml (i.e. elasticsearch)